src/main/java/calebyyy/Ui.java

```diff
@@ -0,0 +1,176 @@
+package calebyyy;
+import java.util.ArrayList;
+
+import calebyyy.tasks.Task;
+
+/**
+ * Represents the user interface of the application.
+ */
+public class Ui {
+    /**
+     * Constructor for Ui.
+     */
+    public Ui() {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Hello! I'm Calebyyy");
+        System.out.println(" What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Print task added message.
+     * @param task
+     * @param taskCount
+     */
+    public void addTaskMessage(Task task, int taskCount) {
+        assert task != null : "Task cannot be null";
+        assert taskCount >= 0 : "Task count cannot be negative";
+        System.out.println("____________________________________________________________");
+        System.out.println(" Got it. I've added this task:");
+        System.out.println("   " + task);
+        System.out.println(" Now you have " + taskCount + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Print goodbye message.
+     */
+    public void byeMessage() {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Bye. Hope to see you again soon!");
+        System.out.println("____________________________________________________________");
```

Comment:
Perhaps you could move the logic into a method that wraps any String in horizontal lines.